object/splitinfo.go

```diff
@@ -35,15 +36,20 @@ func (s *SplitInfo) ToV2() *object.SplitInfo {
 	return (*object.SplitInfo)(s)
 }
 
+// SplitID returns [SplitID] if exists.
```

Comment:
```suggestion
// SplitID returns [SplitID] if it has been set.
```

it is more clear. if an object has been split, split ID _must_ exist